src/Sequelize/CRUD/Update.purs

```diff
@@ -83,6 +86,22 @@ updateModel m a o = do
     handleUndefined x = if isUndefined x then Nothing else Just (unsafeCoerce x)
     -- TODO: check if x really has a runtime representation of Array (Instance a)
 
+updateModel'
+  :: forall a. Model a
+  => ModelOf a
+  -> Options a
+  -> Options a
+  -> Aff { affectedCount :: Int, affectedRows :: Maybe (Array a) }
+updateModel' m a o = do
+  updated <- updateModel m a o
+  case updated.affectedRows of
+    Just rows -> do
+      updatedRows <- pure $ mapInstanceToModel rows
```

Comment:
Only postgres returns rows. Will it throw error of it is running mysql?

Reply:
In those cases, it will go to `Nothing`